.gitignore

```diff
@@ -144,3 +144,5 @@ bm_*.json
 !.vscode/launch.json
 !.vscode/extensions.json
 
+# Clion artifacts
+cmake-build-debug/
```

Comment:
should this be mentioned in the commit message?

Reply:
sure